### PR TITLE
27 members portal

### PR DESCRIPTION
**Pull request by MomentaiChris26**
A new feature that allows members to access a page that contains all their details and provides other information. Also added password field to member's sign up for access to the portal. 

https://trello.com/c/BoRIAk9X/27-member-portal-mp-epic

### Done Checklist

* [ ] Task has been completed as per the Trello card
- Additional work required to complete all items on Trello Card. These items have been split to smaller cards. 
* [ ] All acceptance criteria are met
* [X] Developer has tested this locally
* [X] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [X] Tests have been added or updated
* [ ] Views are responsive for mobile devices
* [ ] Feature files have been written or updated
* [X] Changelog updated

### CODE REVIEW

File: meteor-app/imports/ui/visit/index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const debug = require('debug')('b2b:visit')
 
 export default withTracker(props => {
   const id = props.match.params.id
+  console.log(id)
```
Comment (mikkelking) [inline meteor-app/imports/ui/visit/index.js]: I'm not a fan of console.log, particularly where you are just logging an id like this without any context. If you leave this code in place, the server console will be outputting id's merrily, and you will have no idea where it came from. 

If you really need to be outputting info, you can use `debug()` which does the same, but automatically prefixes the information with the context information, it's also a good idea to provide some local context as well, like this:

```
debug(`memberId: ${id}`)
```

It also won't output anything in production

File: meteor-app/imports/api/constants.js [hunk 2]
```diff
@@ -124,15 +124,15 @@ CONSTANTS.CART_STATUS = {
...
 
 CONSTANTS.DAYS_WEEK = [
-  { id: 0, value: 'Su' },
-  { id: 1, value: 'Mo' },
-  { id: 2, value: 'Tu' },
-  { id: 3, value: 'We' },
-  { id: 4, value: 'Th' },
-  { id: 5, value: 'Fr' },
-  { id: 6, value: 'Sa' }
+  { id: 0, value: 'Sun' },
```
[the remark this answers is not in the document]
Comment (MomentaiChris26) [reply meteor-app/imports/api/constants.js]: The impact is simply a name change in the events page. http://localhost:3030/admin/events It doesn't have any major impacts.

File: meteor-app/imports/api/users/server/publications.js [hunk 3]
```diff
@@ -9,36 +9,70 @@ Meteor.publish('getAllUsers', () => {
...
   addNewUser(func) {
-    const id = Accounts.createUser(
-      {
-        username: 'new user',
-        email: 'newUser@peakadventure.com.au',
-        password: 'changeme'
-      },
-      func
-    )
+    try {
```
[the remark this answers is not in the document]
Comment (MomentaiChris26) [reply meteor-app/imports/api/users/server/publications.js]: I think we should delete this code. Its old code and doesn't seem to be used in any way.

[the remark this answers is not in the document]
Comment (MomentaiChris26) [reply meteor-app/imports/api/users/server/publications.js]: I've deleted it. Doesn't affect anything else in the application.

File: meteor-app/imports/ui/visit/index.js (hunk 1 above)
Comment (MomentaiChris26) [reply meteor-app/imports/ui/visit/index.js]: All good. I've removed the console.logs. I think it was used purely for testing. I'll start using debug() going forward and provide some context.

File: meteor-app/imports/ui/visit/index.js [hunk 4]
```diff
@@ -13,6 +13,7 @@ const debug = require('debug')('b2b:visit')
 
 export default withTracker(props => {
   const id = props.match.params.id
+  console.debug(`memberId: ${id}`)
```
Comment (MomentaiChris26) [inline meteor-app/imports/ui/visit/index.js]: @mikkelking Console.logs removed. This is resolved

**Pull request merged**